src/main/java/frc/robot/arm/ArmSubsystem.java

```diff
@@ -49,17 +48,24 @@ public class ArmSubsystem extends StateMachine<ArmState> {
   private static final InterpolatingDoubleTreeMap CORAL_TX_TO_ARM_ANGLE_TABLE =
       InterpolatingDoubleTreeMap.ofEntries(
           Map.entry(2.66, 5.0), Map.entry(3.89, 0.0), Map.entry(-10.9, -5.0));
+
   private boolean lollipopMode = false;
 
   public void setLollipopMode(boolean lollipopMode) {
     this.lollipopMode = lollipopMode;
     DogLog.log("Arm/LollipopMode", lollipopMode);
   }
 
-  private final MotionMagicVoltage motionMagicRequest =
-      new MotionMagicVoltage(0.0).withEnableFOC(false);
-  private final MotionMagicExpoVoltage autoMotionMagicExpoRequest =
-      new MotionMagicExpoVoltage(0.0).withEnableFOC(false);
+  private static final TrapezoidProfile motionProfile =
```

Comment:
this is not actually a static value. instead of having elevator & arm directly talk to each other, and doing all this kinda icky back & forth with the static fields, just use robotmanager. have arm & elevator expose whatever data they need to robotmanager, and then that can pass in the necessary data to the subsystems.

alternatively, if the flow of data is always arm -> elevator (and not arm <-> elevator), then just pass `ArmSubsystem` into `ElevatorSubsystem` and store it as a field